### DISCUSSION TITLE
Communication between services

### DISCUSSION THREAD

**Discussion by idiotman-2212** [community/community#118697] [Programming Help]
### Select Topic Area

Question

### Body

How to communicate between service order vs product, I tried it and got the result like the photo below:
![image](https://github.com/community/community/assets/82036270/ff04c816-9297-4f57-b573-0637dafe99bf)
When I call the following path http://localhost:9000/api/orders, the order-service and product-service console will have the following image:
![image](https://github.com/community/community/assets/82036270/5a644ade-3ca8-406f-b168-6ef6148a37e5)
![image](https://github.com/community/community/assets/82036270/0b2e363f-3101-4107-939c-58b1dd414645)

This is my code:
![image](https://github.com/community/community/assets/82036270/e9aae6d2-58f8-410d-a76b-2ce5cad465ed)
![image](https://github.com/community/community/assets/82036270/9b8e8648-816b-4201-93bf-7fb2ff2b494b)

I have tried many ways but they still do not display product-service information when communicating with order-service.

**Comment by LiteBrite82** [comment]
Hi @idiotman-2212!

Thanks for posting your most excellent and well documented question! 

Side note: our categories are based on the specific GitHub product the category is named for. For instance Code Search and Navigation is for questions, comments, or concerns with the GitHub Code Search and File Navigation feature function. 

We absolutely have a place for your question, and we are so excited to add it to our "Programming Help" category; I'll move it there for you now. 

Lastly, my take on your question: When I'm not moderating, I am a junior web dev. I'm not familiar with Java yet so I cant be too much help. But shooting from the hip; what database are you using? or is it a 3rd party API? My first instinct is to check the database schemas to make sure the relationships are set up correctly. 

Some things to help other community members answer your question:
* What is your tech stack?
* Can you share the repo?
* Your first image is set to private.

Also, as a shameless GitHub product plug: Have you tried GitHub Copilot? its a game changer! 

Thanks again for being part of our Dev community, @idiotman-2212!

**Reply by idiotman-2212** [reply]
thank for your help, sir
